### PR TITLE
fix add_keyword_plan and generate_keyword_ideas

### PR DESCRIPTION
**Pull request by xflesym**
fix add_keyword_plan and generate_keyword_ideas

### CODE REVIEW

File: examples/planning/add_keyword_plan.rb [hunk 1]
```diff
@@ -38,8 +38,8 @@ def add_keyword_plan(customer_id)
     customer_id,
     plan_campaign,
   )
-  create_keyword_plan_keywords(client, customer_id, plan_ad_group)
-  create_keyword_plan_negative_keywords(client, customer_id, plan_campaign)
+  create_keyword_plan_adgroup_keywords(client, customer_id, plan_ad_group)
```
Comment (wfansh) [inline examples/planning/add_keyword_plan.rb:41]: should this be "create_keyword_plan_ad_group_keywords"?

see line 36

File: examples/planning/add_keyword_plan.rb [hunk 2]
```diff
@@ -104,57 +104,58 @@ def create_keyword_plan_ad_group(client, customer_id, plan_campaign)
   resource_name
 end
 
-def create_keyword_plan_keywords(client, customer_id, plan_ad_group)
-  kp_keyword1 = client.resource.keyword_plan_keyword do |kpk|
-    kpk.text = "mars cruise"
-    kpk.cpc_bid_micros = 2_000_000
-    kpk.match_type = :BROAD
-    kpk.keyword_plan_ad_group = plan_ad_group
+def create_keyword_plan_adgroup_keywords(client, customer_id, plan_ad_group)
```
Comment (wfansh) [inline examples/planning/add_keyword_plan.rb:107]: as above

File: examples/planning/add_keyword_plan.rb (hunk 1 above)
Comment (xflesym) [reply examples/planning/add_keyword_plan.rb]: done.

File: examples/planning/add_keyword_plan.rb (hunk 2 above)
Comment (xflesym) [reply examples/planning/add_keyword_plan.rb]: done

**Pull request merged**